packages/graphql-server/src/resolvers/vendors/__mocks__/vendor.ts

```diff
@@ -0,0 +1,73 @@
+export const vendorMock = {
```

Comment:
Can you add a type annotation to your stubs please - also, these are stubs not mocks, should live in a `__stubs__` folder - thx

Reply:
Hi @willmcvay ,
We'are all putting these files into `__mocks__` folders for all services in graphql-server. Should we move all of them to `__stubs__` now?

Reply:
Ahh, sorry, I must have missed this. Yeah @nphivu414 we should do because the `__mocks__` folder in jest has a special purpose ie if I call `jest.mock('../foo')` it will look for the `foo.ts` file in the `__mocks__` folder. Could get confusing. Stubs really should live in their own folder for clearer separation of concerns.

It's not a big deal though. @tanphamhaiduong do you have a view on this?

Reply:
Hi @willmcvay 
I've just spoken with him about this, it makes sense to move stubs to `__stubs__` folder. @ant066  will fix it for Vendors entity here and I will fix the rest

Reply:
Thanks @nphivu414